packages/ui/src/ux/TextInput.tsx

```diff
@@ -0,0 +1,12 @@
+import { Box, InputLabel, TextField, TextFieldProps } from '@mui/material';
+
+type FormFieldProps = TextFieldProps;
+
+export const TextInput = (props: FormFieldProps) => {
+  return (
+    <Box component="div">
+      <InputLabel htmlFor={props.id}>{props.label}</InputLabel>
+      <TextField {...props} name={props.id} fullWidth />
```

Comment:
```suggestion
      <TextField name={props.id} fullWidth {...props} />
```
In case we'll need short input